src/include/sof/gdb/xtensa-defs.h

```diff
@@ -0,0 +1,18 @@
+#ifndef XTENSA_DEFS_H
+#define XTENSA_DEFS_H
+
+#include <xtensa/specreg.h>
+#include <xtensa/config/core-isa.h>
+#include <xtensa/corebits.h>
+
+#define _AREG0		256
+
+#define STACK_SIZE	1024
+#define DEBUG_PC	(EPC + XCHAL_DEBUGLEVEL)
+#define DEBUG_EXCSAVE	(EXCSAVE + XCHAL_DEBUGLEVEL)
+#define DEBUG_PS	(EPS + XCHAL_DEBUGLEVEL)
+
+/* Enable/Disable GDB Debugging */
+#define GDB_DEBUG 1
```

Comment:
Can we add this to configure.ac for the moment. That way it can easily be added to kconfig menu when it's ready.

Reply:
@lgirdwood, which of these defines exactly would you like to move there?

Reply:
GDB_DEBUG would become CONFIG_GDB_DEBUG

Reply:
@lgirdwood ok, I moved it to configure.ac and renamed as you suggested.